src/libsyntax/ext/build.rs

```diff
@@ -636,4 +636,16 @@ impl<'a> ExtCtxt<'a> {
     pub fn meta_word(&self, sp: Span, w: ast::Name) -> ast::MetaItem {
         attr::mk_word_item(Ident::new(w, sp))
     }
+
+    pub fn meta_nested_word(&self, sp: Span, w: ast::Name) -> ast::NestedMetaItem {
+        attr::mk_nested_word_item(Ident::new(w, sp))
+    }
+
+    pub fn meta_list(&self,
+                     sp: Span,
+                     w: ast::Name,
+                     nested_words: Vec<ast::NestedMetaItem>)
+                     -> ast::MetaItem {
+        attr::mk_list_item(Ident::new(w, sp), nested_words)
+    }
```

Comment:
The `libsyntax/ext/build.rs` is kind of deprecated, so it's better to avoid extending it, but rather use `attr::mk_nested_word_item` and `attr::mk_list_item` directly in the proc macro harness.

Reply:
Got it. I'd love to change it.